WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -5012,6 +5012,7 @@
 					"-l\"sqlite3.0\"",
 					"-l\"z\"",
 				);
+				OTHER_SWIFT_FLAGS = "-D DEBUG";
```

Comment:
I was used to that as well, but the [docs](https://developer.apple.com/library/ios/documentation/Swift/Conceptual/BuildingCocoaApps/InteractingWithCAPIs.html#//apple_ref/doc/uid/TP40014216-CH8-ID34) have the example with a space. Also notice it's `OTHER_SWIFT_FLAGS`. `-DDEBUG` is still being set in `OTHER_CFLAGS`